eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -509,11 +532,13 @@ private class Connector(sourceConnector: SourceConnector) extends Actor {
     case GetReplicationEndpointInfoSuccess(info) if !connected =>
       sourceConnector.links(info).foreach {
         case ReplicationLink(source, target) =>
-          val filter = sourceConnector.connection.filters.get(target.logName) match {
-            case Some(f) => f
-            case None    => NoFilter
+          if (sourceLogIds.forall(_.contains(source.logId))) {
```

Comment:
But they do not have access to the `ReplicationEndpointInfo` that they have to provide to the `links` methods, right? In particular `ReplicationEndpoint.activate` does not have access to an `ReplicationEndpointInfo`, but even in `Acceptor.recovering` the `links` might be across several endpoints. Or am I missing something?